### PR TITLE
Fix complementing zombienet tutorial with terminal outputs

### PR DESCRIPTION
**Pull request by nhussein11**
This PR aims to solve this ticket: https://opslayer.atlassian.net/browse/DRP-45

### CODE REVIEW

**Comment by albertov19** [comment]
@nhussein11 can you address Vale's comments? If this is not relevant anymore please close this PR

**Comment by nhussein11** [comment]
@albertov19, Vale's comments were unnecessary here and have been marked as solved.

I'm waiting for a review here. However, this is not a high priority; it was just something we discussed with Erin and it would be nice to have.

Probably after getting approval here, we can merge it and then port it to the polkadot-docs repo.

**Review by eshaben — APPROVED** [review]
LGTM!

**Pull request merged**
